src/Native/Runtime/thread.cpp

```diff
@@ -33,6 +33,9 @@
 EXTERN_C REDHAWK_API void* REDHAWK_CALLCONV RhpHandleAlloc(void* pObject, int type);
 EXTERN_C REDHAWK_API void REDHAWK_CALLCONV RhHandleFree(void*);
 
+EXTERN_C int (*InitializeRuntimePtr)() = NULL;
```

Comment:
It would look better - more consistent with the overall style of the CoreRT runtime - to keep the pointer static, and expose the setter as a method - like `RhSetRuntimeInitializationCallback` - that is called from the bootstrapper.